arm/apimanagement/apis.go

```diff
@@ -501,33 +429,35 @@ func (client ApisClient) ListByServiceNextResults(lastResults APICollection) (re
 // Update updates the specified API of the API Management service instance.
 //
 // resourceGroupName is the name of the resource group. serviceName is the name
-// of the API Management service. apiID is aPI identifier. Must be unique in
-// the current API Management service instance. parameters is patch parameters.
-// ifMatch is eTag of the API entity. ETag should match the current entity
-// state in the header response of the GET request or it should be * for
-// unconditional update.
-func (client ApisClient) Update(resourceGroupName string, serviceName string, apiID string, parameters PatchParameters, ifMatch string) (result autorest.Response, err error) {
+// of the API Management service. aPIID is aPI identifier. Must be unique in
+// the current API Management service instance. parameters is aPI Update
+// Contract parameters. ifMatch is eTag of the API entity. ETag should match
+// the current entity state in the header response of the GET request or it
+// should be * for unconditional update.
+func (client ApisClient) Update(resourceGroupName string, serviceName string, aPIID string, parameters APIUpdateContract, ifMatch string) (result autorest.Response, err error) {
```

Comment:
`PatchParameters` -> `APIUpdateContract` would be another good thing to call out.